### ISSUE TITLE
docs: identify commnunity supported platforms in getting started documentation

### ISSUE THREAD

**Issue by ryane** [CiscoCloud/mantl#1584]
As we do in the [README](https://github.com/CiscoCloud/mantl#supported-platforms), we should distinguish between supported and community supported platforms in the getting started documentation.

http://docs.mantl.io/en/latest/getting_started/index.html

**Comment by siddharthist** [comment]
@nigerianlegend

To accomplish this, you should:
 1. Check out the master branch with `git checkout master`
 2. Create a new branch with `git checkout -b docs/community-platforms`
 3. Change
```rst
.. toctree::
   :maxdepth: 1

   openstack.rst
   gce.rst
   aws.rst
   digitalocean.rst
   vsphere.rst
   softlayer.rst
   clc.rst
   bare-metal.rst
   vagrant.rst
   triton.rst
```
to
```rst
.. toctree::
   :maxdepth: 1

   openstack.rst
   gce.rst
   aws.rst
   digitalocean.rst
   clc.rst
   vagrant.rst
   triton.rst

Community-supported platforms:

.. toctree::
   :maxdepth: 1

   vsphere.rst
   softlayer.rst
   bare-metal.rst
```
 5. Stage your changes with `git add docs`
 6. Commit your changes with `git commit`
 7. Push your changes to the remote with `git push origin <your-branch>`
 8. Visit https://github.com/CiscoCloud/mantl
 9. [Open a new pull request](https://help.github.com/articles/using-pull-requests/) from your branch, with a short explanation of what you've done.

**Comment by sehqlr** [comment]
Closed via #1654
